gamemodes/terrortown/gamemode/shared/sh_walkspeed.lua

```diff
@@ -0,0 +1,94 @@
+local plymeta = assert(FindMetaTable("Player"), "FAILED TO FIND ENTITY TABLE")
+
+SPEED = SPEED or {}
+
+if SERVER then
+	---
+	-- Sets the initial @{Player} speed multiplier, called in @{GM:TTTBeginRound} after (dumb) players
```

Comment:
These should be two sentences in my opinion. The first being what the function does and then the additional info on where, when and why it is used.